extensions/tasks-parent/quarkus-dev-app/src/main/java/io/resys/thena/tasks/dev/app/TaskResource.java

```diff
@@ -0,0 +1,100 @@
+package io.resys.thena.tasks.dev.app;
+
+/*-
+ * #%L
+ * thena-quarkus-dev-app
+ * %%
+ * Copyright (C) 2021 - 2023 Copyright 2021 ReSys OÜ
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import io.quarkus.vertx.http.Compressed;
+import io.resys.thena.tasks.client.api.TaskClient;
+import io.resys.thena.tasks.client.api.model.Task;
+import io.resys.thena.tasks.client.api.model.TaskCommand;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
```

Comment:
Path naming is a bit convoluted in here.  You need to be very careful when creating a naming scheme for the REST API. 

These have the following problems:
- Repetitive naming
- There should always be a plural: You can't have tasks and then task
- You don't actually have a blobId: You have a taskId

These are currently your final endpoints.

q/tasks/api/active/tasks  
q/tasks/api/task/{blobId}  
q/tasks/api/tasks  
q/tasks/api/task  
q/tasks/api/task/{taskId}  

When naming endpoints, try writing them out first on paper and see how much sense they make.  

This design does not take into account the project scope.  In the current code, for demo purposes, we have hardcoded the project, but in reality, it should be part of the path as well.  So, this also needs to be taken into account.

Let's first try modelling it out on paper.